base_comment_template/models/comment_template.py

```diff
@@ -42,3 +42,10 @@ def _compute_comment_template_ids(self):
                 domain = safe_eval(template.domain)
                 if not domain or record.filtered_domain(domain):
                     record.comment_template_ids = [(4, template.id)]
+
+    def render_comment(self, comment):
```

Comment:
Could you let developer provide _render_template kwargs arguments in order to be able to overwrite `engine='jinja', add_context=None, post_process=False`  )